apmproxy/receiver.go

```diff
@@ -29,10 +29,9 @@ import (
 	"time"
 
 	"github.com/elastic/apm-aws-lambda/accumulator"
-	"github.com/tidwall/gjson"
 )
 
-const txnRegistrationContentType = "application/vnd.elastic.apm.transaction+json"
+const txnRegistrationContentType = "application/vnd.elastic.apm.transaction+ndjson"
```

Comment:
is it OK to break backwards compatibility here? Is there a concern for using older agents? AFAICS, if older agents attempt to send data to this endpoint, we'll just reject it and lose the partial transaction information.

Reply:
This hasn't been implemented yet by any agent, in fact python agent is the first one to start implementing it so backward compatibility is not an issue at this point.